src/borg/helpers.py

```diff
@@ -1781,11 +1802,6 @@ def __init__(self, archive, format, *, json_lines=False):
         for hash_function in hashlib.algorithms_guaranteed:
             self.add_key(hash_function, partial(self.hash_item, hash_function))
         self.used_call_keys = set(self.call_keys) & self.format_keys
-        if self.json_lines:
-            self.item_data = {}
-            self.format_item = self.format_item_json
-        else:
-            self.item_data = static_keys
```

Comment:
needed to move this upwards, so the time formatting gets set up before it is used.